Installation fails due to peerDependencies mismatch

```
[fsevents] Success: "/Users/stomlinson/development/fxa-auth-mailer/node_modules/grunt-nunjucks-2-html/node_modules/nunjucks/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v46-darwin-x64/fse.node" is installed via remote
npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/stomlinson/.nvm/versions/node/v4.3.0/bin/node" "/Users/stomlinson/.nvm/versions/node/v4.3.0/bin/npm" "install"
npm ERR! node v4.3.0
npm ERR! npm  v2.14.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package grunt@1.0.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-copyright@0.2.0 wants grunt@~0.4.2
npm ERR! peerinvalid Peer grunt-bump@0.3.0 wants grunt@>=0.4.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/stomlinson/development/fxa-auth-mailer/npm-debug.log
➜  fxa-auth-mailer git:(master) ✗ npm --version                                             git:(fix-install|✚1
2.14.12
➜  fxa-auth-mailer git:(master) ✗
```

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
   "devDependencies": {
     "eslint-config-fxa": "2.0.0",
     "fxa-auth-db-mysql": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#master",
-    "grunt": "0.4.5",
+    "grunt": "1.0.1",
     "grunt-cli": "0.1.13",
     "grunt-contrib-copy": "0.8.2",
     "grunt-eslint": "17.3.1",

